src/addrman.h

```diff
@@ -517,6 +517,8 @@ friend class CAddrManTest;
             }
         }
 
+        // Attempt to restore the entry's new buckets if the bucket count and asmap
```

Comment:
I find it hard to parse "restore the entry's new buckets".
1. Not only buckets, but also positions.
2. What exactly is "restore" is also a bit unclear, I'm thinking if there's a better verb. Perhaps just "use the serialized [...]"
3.